src/main/java/com/easyrest/pages/OwnerPanel.java

```diff
@@ -31,4 +35,9 @@ public OwnerPanel(WebDriver driver) {
         administrators = new Administrators(driver);
         waiters = new Waiters(driver);
     }
+
+    public void waitForPagePresence() {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(4));
```

Comment:
Hard code

Reply:
I meant seconds) we have this data in constants package, class TimeoutVariable. I recommend taking the task that we have planned Investigate code in project

Reply:
@Zeicks How did you resolve the comment?

Reply:
> I meant seconds) we have this data in constants package, class TimeoutVariable. I recommend taking the task that we have planned Investigate code in project.

Yes, we have TimeoutVariable, but we have not completed them

Reply:
> @Zeicks How did you resolve the comment?
I thought the problem was the locator